### DISCUSSION TITLE
pipes/custompipe/

### DISCUSSION THREAD

**Discussion by giscus** [arungudelli/angularjswiki#196] [Announcements]
# pipes/custompipe/

We can create custom pipes in Angular in two ways.1.Using `ng generate pipe` angular cli command.2. Manually.

https://www.angularjswiki.com/pipes/custompipe/

**Comment by murgesh-ekunde** [comment]
Thanks , it helped me in solving my  assignment!

**Reply by arungudelli** [reply]
Glad it helped
